dataline-config/init/src/main/resources/config/DESTINATION_CONNECTION_SPECIFICATION/8442ee76-cc1d-419a-bd8b-859a090366d4.json

```diff
@@ -0,0 +1,27 @@
+{
+  "destinationId": "8be1cf83-fde1-477f-a4ad-318d23c9f3c6",
+  "destinationSpecificationId": "8442ee76-cc1d-419a-bd8b-859a090366d4",
+  "specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Singer CSV Target Spec",
+    "type": "object",
+    "required": [],
+    "additionalProperties": false,
+    "properties": {
+      "delimiter": {
+        "description": "Delimiter used to separate fields.",
+        "type": "string",
+        "examples": [","]
+      },
+      "quotechar": {
+        "description": "What character in the input should be treated as a quotation mark.",
+        "type": "string",
+        "examples": ["'"]
+      },
+      "destination_path": {
+        "description": "Path to write the data. The provided directory _must_ exist on the filesystem. This path will be concatenated onto the `LOCAL_ROOT`. e.g. If `LOCAL_ROOT = /tmp/dataline_local` and destination_path is `cars/models`, then the directory `/tmp/dataline_local/cars/models` must already exist.",
```

Comment:
Is there a reason the entrypoint shouldn't create the directory if it doesn't exist, just to make it easier and all in-app? 

It's also worth saying explicitly that `/tmp/dataline_local` is the default path. Really it'd be best if it said exactly what the actual local root was, but that's hard to do at this point...

Reply:
i'll add it.

Reply:
good point. i can't think of a reason not to create the dir.